README.md

```diff
@@ -711,6 +711,8 @@ arXiv, 2020.
 IEEE Trans. Image Processing, 30:2488-2500, 2021. [Project page](https://github.com/RodrigoGantier/SPADE_E2VID)
 - [Baldwin et al., arXiv 2021](#Baldwin21arxiv).  
 *Time-Ordered Recent Event (TORE) Volumes for Event Cameras*.
+- [Muglikar et al., CVPRW 2021](#Muglikar21cvprw).  
```

Comment:
Due to the methodology that uses image reconstruction, I added it to this section too.

Reply:
I would remove it from here. It is not a method for image reconstruction; it is just using it for something else. Otherwise, we would have to put all papers that use E2VID and similar in this category...